packages/react-shopify-app-bridge/src/Provider.tsx

```diff
@@ -266,17 +280,35 @@ type ProviderLocation = {
   asPath?: string;
 };
 
-export const Provider = ({ type, children, api }: { type?: AppType; children: ReactNode; shopifyApiKey: string; api: AnyClient }) => {
+export const Provider = ({
+  type,
+  children,
+  api,
+  location: _location,
```

Comment:
Are there any doc comments that need updating anywhere

Reply:
I added some comments to the two new optional parameters. Let me see if there are docs on the gadget side.